client/src/main/java/org/spine3/base/Commands.java

```diff
@@ -53,7 +52,7 @@
     /**
      * A substring which the {@code .proto} file containing commands must have in its name.
      */
-    public static final String COMMANDS_FILE_SUBSTRING = "commands";
+    public static final String COMMANDS = "commands";
```

Comment:
If we name it `suffix` we have to check the file name with `endsWith()` method. But now it is checked via `contains()` method, for more flexibility (maybe, a client wants to name the file `commands_order` instead of `order_commands` or just `commands`).